marklogic-data-hub-central/ui/src/components/common/hc-button/hc-button.tsx

```diff
@@ -0,0 +1,26 @@
+
+import React from "react";
+import {Button, ButtonProps, Spinner} from "react-bootstrap";
+
+export interface HCButtonProps extends ButtonProps {
+  loading?: boolean
+
+}
+
+const HCButton: React.FC<HCButtonProps> = ({loading, children, ...others}) => {
+  return (
+    <Button data-testid="hc-button-component" disabled={loading ?? loading} {...others}>
```

Comment:
Won't there be a an issue if we have more than one button and they have the same data-testid?

Reply:
@aniamy  you can override the inner property by passing it a new data-testid, in the component where you will use the button